modules/ipi-install-configuring-networks.adoc

```diff
@@ -0,0 +1,9 @@
+// This is included in the following assemblies:
+//
+// ipi-install-prerequisites.adoc
+
+[id='configuring-networks_{context}']
+
+= Configuring networks
+
+Documentation for customizing the network configuration is out-of-scope for this release. For details on customizing the network configuration, see link:https://access.redhat.com/solutions/5460671[Customizing network configuration for OCP 4.6 bare metal IPI installation].
```

Comment:
This link is not active; If there's no alternate resource we can link to, I recommend removing this file until we have content in a future release.

Reply:
The link is active for me. You're probably not logged in to access.redhat.com.

Reply:
Yeah, it's loading now, but saying it's a draft and not yet published. 
I do think we should remove the first sentence about docs being out of scope. We typically don't mention that it our doc set.

Reply:
Removed the sentence and sent an email to jlebon@redhat.com regarding publishing the article.